src/wazuh_mcp_server/scripts/test_connection.py

```
@@ -81,7 +81,7 @@ async def test_connection():
                 else:
                     print("  ℹ Using Server API for alerts")
                     
-            except Exception as e:
+            except Exception:
                 print(f"⚠ Alert query failed: {str(e)}")
```

Ruff (0.11.9)

Undefined name `e`

(F821)